src/components/GlossaryPage.css

```
@@ -63,6 +63,42 @@
+  padding: 20px 0 0;
+}
+.glossary-breadcrumb ol {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 8px;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  font-size: 12.5px;
+  font-family: ui-monospace, "SF Mono", "Menlo", monospace;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)